### DISCUSSION TITLE
Awaiting for the new updated Waterfox 6.5.0, Please patch it with KDE Global Menu Patch (Canonical)

### DISCUSSION THREAD

**Discussion by dssharma2309** [hawkeye116477/waterfox-deb-rpm-arch-AppImage#86] [Ideas]
Hi, Really awaiting for the release of the patched version of new Waterfox 6.5.0, hope to have KDE Global Menu Patch (Canonical Patch)

**Comment by hawkeye116477** [comment]
Hello. That feature is already added by Waterfox team. However that may be last or almost last release of Waterfox from me. Future releases likely will be for Floorp with added Global Menu patch :smile:.

**Reply by ghost** [reply]
the global menu is my only deal breaker with firefox based browsers. your waterfox with gobal menu patch was my reason to move to waterfox. if you move to floorp, I'll move there too. 
currently waterfox lost the global menu support.

**Reply by dssharma2309** [reply]
> Hello. That feature is already added by Waterfox team. However that may be last or almost last release of Waterfox from me. Future releases likely will be for Floorp with added Global Menu patch 😄.

I dont think Waterfox team has updated the Global menu in the new 6.5.1 version, anyways if you are working on Floorp with the same patch that's even better, Floorp is very logical switch much better cycle with main Firefox.  Thanks for the update and looking forward to the Floorp version, Best Wishes.

**Reply by hawkeye116477** [reply]
> I dont think Waterfox team has updated the Global menu in the new 6.5.1 version, 

It works but only for X11. Simply that's old patch, made by Canonical. New solution for Wayland is being prepared by Mozilla itself.
